pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -403,10 +403,19 @@ func (this *SnapshotManager) deleteSnapshot(peID astrolabe.ProtectedEntityID, ba
 			log.WithError(err).Errorf("DeleteSnapshot: Failed to lookup the env variable for velero namespace")
 			return err
 		}
-		snapIDDecoded, err := decodeSnapshotID(peID.GetSnapshotID(), this.FieldLogger)
-		if err != nil {
-			log.WithError(err).Errorf("Failed to retrieve decoded snapshot id to search for on-going uploads, original pe-id :%s", peID.String())
-			return err
+
+		var snapIDDecoded string
+		if peID.GetPeType() == "ivd" {
+			log.Infof("The pe-id %s is not encoded, detected deletion of backup from plugin <=1.0.2", peID.String())
```

Comment:
Just by checking this, we can be sure this is from plugin <= 1.0.2?

Reply:
I think it is safer to change the message to "The pe-id %s is not encoded. This maybe a request to delete the backup from plugin <=1.0.2"

Reply:
can we change `plugin <= 1.0.2` to `plugin prior or equal to v1.0.2`